base/uk.ac.stfc.isis.ibex.e4.client.product/ibex.product

```diff
@@ -20,6 +20,8 @@
 -Dpython.console.encoding=UTF-8
 -Dpython.import.site=false
 -XX:+CreateCoredumpOnCrash
+-XX:HeapDumpPath=C:/Instrument/Scratch/CrashDumps/ibexcd%p.hprof
+-XX:ErrorFile=C:/Instrument/Scratch/CrashDumps/ibexerror%p.log
```

Comment:
Maybe `ibex_client_%p.hprof` and `ibex_client_%p.log` to make it clearer it is the client